### ISSUE TITLE
Will be used to backup OgarUnlimited

### ISSUE THREAD

**Issue by Agarian** [Agarian/Ogar-unlimited#1]

**Comment by Megabyte918** [comment]
Can you come on discord? @Agarian

**Comment by Megabyte918** [comment]
I think we should start over from version 16.3.5, because it does not have any lag (kinda). Then we could work our way up with re-adding all the features.

**Comment by Agarian** [comment]
True, but my edited version has 17.4.0 and i don't wanna revert :(
